subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/provider/ClassPathModeExceptionCollector.kt

```diff
@@ -16,22 +16,20 @@
 package org.gradle.kotlin.dsl.provider
 
 import org.gradle.internal.concurrent.Stoppable
+import org.gradle.internal.service.scopes.ExceptionCollector
 
 
-open class ClassPathModeExceptionCollector : Stoppable {
-
-    private
-    val collection = mutableListOf<Exception>()
+open class ClassPathModeExceptionCollector(val delegate: ExceptionCollector) : Stoppable {
```

Comment:
The ctor argument should be private
```suggestion
open class ClassPathModeExceptionCollector(private val delegate: ExceptionCollector) : Stoppable {
```

Reply:
ty